codingtest_ajw/7일차/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,94 @@
+// 처음에 if문으로 구매한 로또배열이 win_nums배열안에 있는지 검사해서
+// 안에 존재하면 count값을 증가시키고 만약 구매한 로또 배열에 0이 있으면
+// zeroCount값을 증가시킨다음에 count값이 6개면 다 맞았기 때문에 결과값에 1,1 을 넣어주고
+// count값이 5개면 나머지 하나 값이 0인지 그냥 틀린 값인지 검사해서 0이면 1,2 를 틀린값이면 2,2를 결과값을 출력한다.
+// 이것을 count 6...0까지 zeroCount 값이 0...6까지 나올 수 있는 모든 경우의 수를 검사한다.
+
+function solution(lottos, win_nums) { 
+    let answer = [];
+    let count = 0;
+    let zeroCount = 0;
+    for (let i=0; i<lottos.length; i++){
```

Comment:
```suggestion
    for (let i=0; i<6; i++){
```
**문제의 제한사항- 1번 항목입니다. [lottos는 길이 6인 정수 배열입니다.]**